test/statmodels.jl

```diff
@@ -100,15 +103,17 @@ end
 @test TestStat(TestStat(0.05)) === TestStat(0.05)
 @test isless(TestStat(0.01), 0.05)
 @test isless(TestStat(0.01), NaN) == isless(0.01, NaN)
+@test isless(0.01, TestStat(NaN)) == isless(0.01, NaN)
 @test (TestStat(0.01) < NaN) == (0.01 < NaN)
 @test isless(NaN, TestStat(0.01)) == isless(NaN, 0.01)
 @test (NaN < TestStat(0.01)) == (NaN < 0.01)
 @test isequal(TestStat(0.01), NaN) == isequal(0.01, NaN)
 @test (TestStat(0.01) == NaN) == (0.01 == NaN)
 @test isequal(NaN, TestStat(0.01)) == isequal(NaN, 0.01)
 @test (NaN == TestStat(0.01)) == (NaN == 0.01)
+@test isequal(PValue(NaN), NaN) == isequal(NaN, NaN)
+@test (PValue(NaN) == NaN) == (NaN == NaN)
```

Comment:
```suggestion
@test isequal(TestStat(NaN), NaN) == isequal(NaN, NaN)
@test (TestStat(NaN) == NaN) == (NaN == NaN)
```

Reply:
Woops. Copy-paste is hard, but that's actually a clue that I should just make this into a big loop....